packages/menu-bar/test/dom/menu-bar.test.js

```diff
@@ -50,4 +55,15 @@ describe('menu-bar', () => {
     await nextRender();
     await expect(menu._subMenu.$.overlay).dom.to.equalSnapshot(SNAPSHOT_CONFIG);
   });
+
+  it('overflow', async () => {
```

Comment:
Looking at the code base, we currently stick to an unofficial rule of preferring unit tests to snapshot tests if they involve many steps or they toggle something. Here is an example:

https://github.com/vaadin/web-components/blob/25db6771b3128cce504ba9bdf7c69cc2d8f0dd30/packages/select/test/accessibility.test.js#L22-L27